### ISSUE TITLE
Adjust public link upload tests

### ISSUE THREAD

**Issue by phil-davis** [owncloud/core#39758]
Reva issue https://github.com/cs3org/reva/issues/2480 is fixing up the upload behavior in public links. Rather than appending (2) (3) etc to the filename to generate unique file names, that proposes to add a "timestamp" in some form to the filename.

PR https://github.com/cs3org/reva/pull/2510 is currently in development, and there also discussion there.

When the requirement has been decided, skip scenarios like https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiSharePublicLink2/uploadToPublicLinkShare.feature#L24 on oCIS and reva, and add scenarios for oCIS and reva only that test the new requirement.

**Comment by phil-davis** [comment]
See https://github.com/cs3org/reva/issues/2480#issuecomment-1046446517

This is still blocked waiting for a decision on the naming format.

**Comment by phil-davis** [comment]
Update: that issue https://github.com/cs3org/reva/issues/2480 is still open. PR https://github.com/cs3org/reva/pull/2510 seems to have stalled.

So this test-refactoring issue remains blocked.

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This is an internal QA test-refactoring task (adjust public-link upload acceptance tests for oCIS/reva), not a user-facing bug or feature request. It was explicitly placed in the **Blocked** column in 2022, waiting on a naming-format decision in the upstream reva work:

- reva issue [cs3org/reva#2480](https://github.com/cs3org/reva/issues/2480) is now **closed**.
- reva PR [cs3org/reva#2510](https://github.com/cs3org/reva/pull/2510) was **closed without being merged** (it stalled, as noted in the last comment here).

So the upstream requirement this test work depended on never landed, and `owncloud/core` is now in maintenance mode (the referenced `uploadToPublicLinkShare.feature` path no longer resolves in the repo). The original blocker is therefore moot and this test-adjustment work is no longer actionable here.

Is this still relevant to anyone? If not, it can likely be closed. If the public-link upload test behavior still needs adjusting for a current oCIS/reva release, please track it in the relevant active repository with up-to-date details.

**Comment by phil-davis** [comment]
This is no longer relevant. No different behaviour was decided. So there are no tests to add/adjust.
